css/styles.css

```diff
@@ -21,3 +21,49 @@
   color: #f65239;
   background-color: rgba(0, 0, 0, .25);
 }
+
+
+/* Carousel Styles Start */
+.item {
+  height: 450px;
+  background-size: cover;
+  background-position: center;
+}
+
+.wood-bg{
+  background-image: url('../img/wood-2.jpg');
+}
+.seating-bg {
+  background-image: url('../img/restaurant-2.jpg');
+}
+
+.farm-bg {
+  background-image: url('../img/farm-2.jpg')
+}
+
+.hero-logo {
+  width: 240px;
+}
+
+.flex-slide {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+}
+
+.slide-text {
+  margin-bottom: 50px;
+}
+
+.btn-warning {
+  background-color: #F8512F;
+  border-color: transparent;
+}
+
+.btn-warning.active, .btn-warning.focus, .btn-warning:active, .btn-warning:focus, .btn-warning:hover, .open>.dropdown-toggle.btn-warning {
```

Comment:
I'm pretty sure this will work by just writing .btn-warning:hover { } Let me know if I'm wrong because I'm fairly certain that's what I did

Reply:
I may have just been unnecessarily specific but I'm sure they both work.